openedx/core/djangoapps/kafka_consumer/management/commands/consume_events.py

```diff
@@ -0,0 +1,94 @@
+"""
+Management command for listening to license-manager events and logging them
+"""
+
+import logging
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from confluent_kafka import KafkaError, KafkaException, DeserializingConsumer
+from confluent_kafka.schema_registry import SchemaRegistryClient
+from confluent_kafka.serialization import StringDeserializer
+from openedx.core.djangoapps.kafka_consumer.consumers import getHandler
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Listen for license events from the event bus and log them. Only run on servers where KAFKA_ENABLED is true
+    """
+    help = """
+    This starts a Kafka event consumer that listens to the specified topic and logs all messages it receives. Topic
+    is required.
+
+    example:
+        manage.py ... consume_events -t license-event-prod
```

Comment:
1. I don't understand the question. Currently, license-manager is sending events to the 'license-event-prod' topic. the event keys are the original segment keys (eg "edx.server.license-manager.license-lifecycle.activated")
2. I'm going to put this one in a todo since I'm not actually sure about the way we're going to do that. 
3. Will fix
4. Given that we only had one cluster, instead of using the same topic name on dev, stage, and prod, i opted to change the topic name so we could distinguish between environments. eventually, i was assuming we'd have a separate stage cluster and mostly use devstack/tutor for dev work.